test/e2e/proxy_test.go

```diff
@@ -76,10 +75,25 @@ func TestProxy(t *testing.T) {
 		return false, nil
 	})
 
-	for _, err = range framework.CheckEnvVars(proxyEnvVars["clusterVars"], newConsoleDeployment.Spec.Template.Spec.Containers[0].Env, true) {
-		t.Fatal(err)
-	}
-
+	counter := 0
+	maxCount := 60
```

Comment:
So this simply takes a while to propagate correct?

Reply:
Ready to remove the `[WIP]`?

Reply:
> So this simply takes a while to propagate correct?
yes